en/exercises/guided_tour/physical_security_survey.md

```diff
@@ -0,0 +1,77 @@
+### Physical Security Survey
+
+
+Do you have policies and procedures for authorizing and limiting unauthorized physical access to digital systems and the facilities in which they are housed? 
+
+  - [ ] No
+  - [ ] Yes
+
+
+Do your policies and procedures specify the methods used to control physical access to your secure areas, such as door locks, access control systems, security officers, or video monitoring? 
+
+  - [ ] No
+  - [ ] Yes
+
+If yes, what are there?
+
+Is access to your computing area controlled (e.g. reception or security desk, sign-in/sign-out log, temporary/visitor badges etc)? 
+
+  - [ ] No
+  - [ ] Yes
+
+Are visitors escorted into and out of controlled areas?
+
+  - [ ] No
+  - [ ] Yes
+
+If yes, who is responsible for it?
+
+Are your workstations inaccessible to unauthorized users (e.g. located away from public areas)?
+
+  - [ ] No
+  - [ ] Yes
+
+
+Is your computing area and equipment physically secured? 
+  - [ ] No
+  - [ ] Yes
+ 
+If yes, how is it secured?
+
+Are there procedures in place to prevent computers from being left in a logged on state, however briefly? 
+
+  - [ ] No
+  - [ ] Yes
+
+If yes,  what are the procedures?
+
+Do you have procedures for protecting data during equipment repairs? 
+
+  - [ ] No
+  - [ ] Yes
+
+If yes, what are the procedures?
+
+Do you have policies covering laptop security (e.g. cable lock or secure storage)? 
```

Comment:
We have to wrote the content using our words or reference the content accordingly I will refer to @joncamfield on this
https://books.google.com/books?id=D5w3BwAAQBAJ&pg=PA197&lpg=PA197&dq=Do+you+have+policies+covering+laptop+security+(e.g.+cable+lock+or+secure+storage)?&source=bl&ots=VO6hjtpYr0&sig=SQvZkkdnPdSl2C4bQ6vkIJde0xU&hl=en&sa=X&ved=0ahUKEwiFg4GMk_bXAhWyQt8KHXlJACcQ6AEIJzAA#v=onepage&q=Do%20you%20have%20policies%20covering%20laptop%20security%20(e.g.%20cable%20lock%20or%20secure%20storage)%3F&f=false

Reply:
As I use this tool quite often and found it to be simple. I will go with writing something similar but with our words. Advise on if I should proceed or we create reference only